### PR TITLE
Add `media-src` support to CSP and allow https://news.files.bbci.co.uk

### LINKED ISSUE #7602: Update `media-src` CSP
**Is your feature request related to a problem? Please describe.**
Sometimes VJ includes may load .mp3's from the `https://news.files.bbci.co.uk` domain.

**Describe the solution you'd like**
Update the CSP header to include `https://news.files.bbci.co.uk/` in `media-src`

**Describe alternatives you've considered**

**Testing notes**
- No testing required

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by joshcoventry**
Resolves https://github.com/bbc/simorgh/issues/7602

**Overall change:**
Adds support for media-src to our CSP
Adds Update the CSP header to include https://news.files.bbci.co.uk/ in media-src

**Code changes:**

- `media-src` block added to CSP
- Tests updated

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by chris-hinds — APPROVED** [review]
LGTM 👍

File: src/server/utilities/cspHeader/index.js [hunk 1]
```diff
@@ -348,6 +348,20 @@ const directives = {
       'https://static.bbci.co.uk', // STY includes
     ],
   },
+  mediaSrc: {
+    ampLive: [
+      'https://news.files.bbci.co.uk', // VJ includes may load .mp3 files
```
Comment (rhenshaw56) [inline src/server/utilities/cspHeader/index.js:357]: are the .mp3 files also loaded on amp if those are rendered in an amp iframe ?

**Review by rhenshaw56 — APPROVED** [review]

**Review by simonsinclair — APPROVED** [review]

File: src/server/utilities/cspHeader/index.js [hunk 2]
```diff
@@ -348,6 +348,20 @@ const directives = {
...
+    ],
+    canonicalLive: [
+      'https://news.files.bbci.co.uk', // VJ includes may load .mp3 files
+    ],
+    ampNonLive: [
+      'https://news.files.bbci.co.uk', // VJ includes may load .mp3 files
```
Comment (andrewscfc) [inline src/server/utilities/cspHeader/index.js:363]: Should the non-live be `https://news.test.files.bbci.co.uk` @eagerterrier ?

Comment (jimjohnsonrollings) [reply src/server/utilities/cspHeader/index.js:363]: Presumably we need the live one on test too/regardless, as we can render live content on our test env?

Comment (eagerterrier) [reply src/server/utilities/cspHeader/index.js:363]: I think I agree with Jim. If we have that ability it would be really handy to preview live on test

**Review by andrewscfc — APPROVED** [review]

**Pull request merged**